Web/starter_project_iconics/components/team/TeamCard.tsx

```diff
@@ -0,0 +1,37 @@
+
+export interface TeamCardProps{
+    name: string
+    job: string
+    description: string
+    instagram?: string
+     linkedin?: string
+     facebook?: string
+     avatar?: string
+}
+
+function TeamCard({name, job, description, avatar = 'https://www.dmarge.com/wp-content/uploads/2021/01/dwayne-the-rock-.jpg', instagram, linkedin, facebook}: TeamCardProps) {
```

Comment:
Why you need the avatar value here, not good practice, also you doubled it.

Reply:
ok

Reply:
I have removed it, I wanted to set it as the default value